api/src/models/massageLaw.js

```diff
@@ -0,0 +1,13 @@
+/** Schema representing a massage law **/
+const mongoose = require('mongoose');
+
+const MassageLaw = new mongoose.Schema({
+  state: { type: String, required: false },
+  city: { type: String, required: false },
+  strengthOfStateLaw: { type: String, required: false },
+  codeLink: { type: String, required: false },
```

Comment:
sorry they just got back on this!! we can remove both codeLink and codeSection because they don't find it useful for the public to access